model/service.go

```diff
@@ -54,6 +54,9 @@ type Service struct {
 	// service DNS name.  External services are name-based solution to represent
 	// external service instances as a service inside the cluster.
 	ExternalName string `json:"external"`
+
+	// Annotations on the service.
+	Annotations map[string]string `json:"annotations,omitempty"`
```

Comment:
Does this have to be exposed? This forces all service registries to have annotations on services? Isn't this just an implementation detail on k8s?